base/linalg/lu.jl

```diff
@@ -313,8 +314,9 @@ end
 
 inv!(A::LU{<:BlasFloat,<:StridedMatrix}) =
     @assertnonsingular LAPACK.getri!(A.factors, A.ipiv) A.info
-inv(A::LU{<:BlasFloat,<:StridedMatrix}) =
-    inv!(LU(copy(A.factors), copy(A.ipiv), copy(A.info)))
+inv!(A::LU{T,<:StridedMatrix}) where {T} =
+    @assertnonsingular A_ldiv_B!(A.factors, copy(A), eye(T, size(A,1))) A.info
+inv(A::LU{<:Any,<:StridedMatrix}) = inv!(copy(A))
```

Comment:
there's a `copy(A)` on this line, then another `copy(A)` in the `inv!(A::LU{T,<:StridedMatrix})` that it calls, right?